_assets/scss/elements/_typography.scss

```diff
@@ -78,6 +78,17 @@
   @include breakpoint($large) {}
 }
 
+@mixin paragraph {
```

Comment:
I'm checking with Andre on that tomorrow, since line-height is actually different between h4 and paragraph. Modified everything else in the meantime